docs/source/opentelemetry.mdx

```diff
@@ -0,0 +1,214 @@
+---
+title: OpenTelemetry in Apollo Server
+sidebar_title: OpenTelemetry
+---
+
+import {
+  ExpansionPanel,
+} from 'gatsby-theme-apollo-docs/src/components/expansion-panel';
+
+[OpenTelemetry](https://opentelemetry.io/) is a collection of open-source tools for generating and processing telemetry data (such as logs and metrics) from different systems in a generic, consistent way.
+
+You can configure your gateway, your individual subgraphs, or even a monolothic Apollo Server instance to emit telemetry related to processing GraphQL operations.
+
+Additionally, the `@apollo/gateway` library provides built-in OpenTelemetry instrumentation to emit [gateway-specific spans](#gateway-specific-spans) for operation traces.
+
+> Apollo Studio does _not_ currently consume OpenTelemetry-formatted data. To push trace data to Studio, see [Federated trace data](./metrics/).
+>
+> You should configure OpenTelemetry if you want to push trace data to an OpenTelemetry-compatible system, such as [Zipkin](https://zipkin.io/) or [Jaeger](https://www.jaegertracing.io/).
+
+## Setup
+
+### 1. Install required libraries
+
+To use OpenTelemetry in your application, we recommend installing _at least_ the following `@opentelemetry` Node.js libraries (you can install additional instrumentation libraries as desired):
+
+```bash
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/node@0.22 \
```

Comment:
I found myself needing to hard-set these version requirements because some libraries had not yet set `0.22` as their "current" release, and it caused hiccups